lib/pyfrc/physics/units.py

```diff
@@ -0,0 +1,25 @@
+'''
+    Useful constants that can be used to convert between various units. To use
+    these, multiply the value you wish to convert by the given constants
+'''
+
+#: Length: Meters -> Feet
+M_TO_FT = 3.2808398950131
+
+#: Length: Meters -> Inches
+M_TO_IN = 39.3701
```

Comment:
Wouldn't it be better to define these ones as the reciprocal of the below (which are exact values)?